Header header
int8 state
int8 target_aruco_ID
string active_valve_block
int8 ball_valve
int8 reset_levelling
